manifests/cf-manifest/deployments/060-aws-broker.yml

```diff
@@ -0,0 +1,61 @@
+releases:
+  - name: aws-broker
+    version: 0.0.1
+
+jobs:
+  - name: rds_broker_z1
+    instances: 1
+    resource_pool: rds_broker_z1
+    templates:
+      - name: rds-broker
+        release: aws-broker
+    properties:
+      rds-broker:
+        aws_access_key_id: ""
+        aws_secret_access_key: ""
+        aws_region: "eu-west-1"
+        password: (( grab secrets.rds_broker_admin_password ))
+        catalog:
+          services:
+            - id: "ce71b484-d542-40f7-9dd4-5526e38c81ba"
+              name: "rds"
+              description: "AWS RDS PostgreSQL service"
+              bindable: true
+              tags:
+                - "postgres"
+                - "relational"
+              metadata:
+                displayName: "AWS RDS Postgres"
+                imageUrl: ""
+                longDescription: "AWS RDS postgres service"
+                providerDisplayName: "Amazon Web Services"
+                documentationUrl: "https://aws.amazon.com/documentation/rds/"
+                supportUrl: "https://forums.aws.amazon.com/forum.jspa?forumID=60"
+              plan_updateable: true
+              plans:
+                - id: "5b8282cf-a669-4ffc-b426-c169a7bbfc71"
+                  name: "5.6-medium"
```

Comment:
I think it's right to name it after the Postgres version (not MySQL 😉 ) and instance size, because those are the differentiating properties of this plan.